src/frame/js/actions/auth.js

```diff
@@ -1,33 +1,85 @@
+import { batchActions } from 'redux-batched-actions';
+
 import http from './http';
-import { handleUserConversationResponse } from './conversation';
+import { resetConversation, handleUserConversationResponse, disconnectFaye } from './conversation';
+import { setUser, resetUser } from './user';
+import { resetIntegrations } from './integrations';
 
-import { getClientInfo } from '../utils/client';
+import { getClientId, getClientInfo } from '../utils/client';
 import { removeItem } from '../utils/storage';
 
 export const SET_AUTH = 'SET_AUTH';
 export const RESET_AUTH = 'RESET_AUTH';
 
-export function login() {
+export function login(userId, jwt) {
     return (dispatch, getState) => {
-        const {config: {appId}, user: {userId}, auth: {sessionToken}} = getState();
+        const {config: {appId}, auth: {sessionToken}} = getState();
+
+        const actions = [
+            setAuth({
+                jwt
+            }),
+            setUser({
+                userId
+            }),
+            resetConversation(),
+            resetIntegrations()
+        ];
+
+        dispatch(disconnectFaye());
```

Comment:
`batchActions` doesn't work for async actions like `disconnectFaye`. At least I couldn't get it to work